travis_build_test.sh

```diff
@@ -13,6 +13,7 @@ end eslint
 
 start compile
 npm run compile
+[ -e dist.zip ] || die 'Missing dist.zip'
```

Comment:
Maybe say `Missing dist.zip, please check that it was produced by npm compile in build.sh`

Reply:
I 😍
@mccalluc other than that please merge